### DISCUSSION TITLE
Does Filament support 2FA (or in future) ?

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#1382] [Help] [answered]
Does Filament support 2FA (or planned in future) ?
2FA Codes, SMS through external sms provider for example.

**Comment by ryangjchandler** [comment, marked as answer]
@zenati Filament doesn't have any support for 2FA out of the box and it's not on our roadmap to support in the future. If you're looking for 2FA functionality, you can quite easily replace the default Login page with your own.

The login page we provide is more of a "getting started" convenience. Chances are you'll use your own auth flow since we don't even have registration/reset password flows.
